README.md

```diff
@@ -37,8 +37,21 @@ git clone https://github.com/archesproject/arches-qgis.git
 3. Head to the QGIS Plugins tab and select "Manage and Install Plugins".
 4. Search for and select "Arches Project" from the list of all plugins.
 
-## Tips for developing
+## Information for developers
 If you wish to develop with the QGIS Arches plugin, below are some helpful tips that will help, and make life easier.
 - Installation via GitHub is the easiest method to develop.  This can be done by git cloning in the plugins path (shown above) and (optionally) creating a symbolic link to somewhere much easier to find e.g. your home directory.
 - The QGIS plugin "Plugin Reloader" is incredibly useful for reloading plugins to reflect code changes.  This can be found on the QGIS plugins repository, and configured to reload specific plugins with Ctrl+F5.
-- If you wish to develop UI elements, you'll need Qt Creator installed: https://doc.qt.io/qtcreator/.
+
+### Developing the user interface
+QGIS uses PyQt as the framework for UI, specifically Qt 5.15.
+
+If you wish to develop UI elements for the arches-qgis plugin you'll need to install [Qt Creator](https://doc.qt.io/qtcreator/), an IDE for Qt applications.   
+It's recommended to install Qt and QtCreator using the online installer found here: https://www.qt.io/download-qt-installer-oss. Bundled is a Maintenance Tool that makes updating, and installing/uninstalling additional components very easy.  
+Offline, version-specific packages for Qt and QtCreator can be downloaded here: https://www.qt.io/offline-installers.
+
+QtCreator uses Qt6 out the box, so you'll need to use the Qt Mainentance Tool to install the archived version of Qt 5.15.
+Expand Qt 5.15.2 and ensure only MinGW 8.1.0 is checked.
+Once installed, add the new version of Qt as a kit in the QtCreator preferences, see the documentation for more information: https://doc.qt.io/qtcreator/creator-targets.html.
```

Comment:
I don't see an option to download earlier version of Qt. Maybe I skipped a step?

<img width="1432" height="999" alt="image" src="https://github.com/user-attachments/assets/fc174f53-452c-4a37-9618-506d2547cbde" />